cmd/podman/containers/stats.go

```diff
@@ -146,7 +146,9 @@ func stats(cmd *cobra.Command, args []string) error {
 func outputStats(reports []define.ContainerStats) error {
 	headers := report.Headers(define.ContainerStats{}, map[string]string{
 		"ID":            "ID",
+		"UpTime":        "CPU TIME (S)",
```

Comment:
Is UpTime really correect? I've always heard that as the total duration a system has been running for, not cumulative CPU time?

Reply:
This is the result of podman stats 
```
ID            NAME            CPU TIME (S)     CPU %   AVG CPU %  MEM USAGE / LIMIT  MEM %   NET IO   BLOCK IO  PIDS
f10ae061aa6b  thirsty_colden  14m1.261643561s  1.28%   6.60%      729.4MB / 16.02GB  4.55%   -- / --  -- / --   117
```
I made it so that the CPU time pulls the container creation time if the variable is time's zero value. I assumed that CPU time just basically meant up time of the container? If I was wrong I can go back and adjust accordingly.

Reply:
No, CPU time is the number of seconds or CPU time the container has actively used, not seconds it has been alive.

Reply:
@mheon I _think_ most recent commit fixed this. I didn't realize cgroupStats.CPU.Usage.Total was a duration in ns of how much up time the group had. I switched around some things with avg CPU % given this information as well.